src/meteofrance_api/model/observation.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""Weather observation Python model for the Météo-France REST API."""
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict  # pylint: disable=no-name-in-module
+else:
+    from typing_extensions import TypedDict
+	
+class Observation:
+	def __init__(self, data: TypedDict):
```

Comment:
Your model is nice, but other are not built like that.
Could you make this model as the others ?

And also add a doc.

Reply:
I commited an update (new model + doc).
Please review whenever you can.